<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Jumps</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLineEdit" name="NameEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="SurnameEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="ResultEdit"/>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="Universities">
            <property name="minimumSize">
             <size>
              <width>188</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>NULP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LNU</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Result:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>University:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>751</width>
            <height>0</height>
           </size>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>188</number>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Ім'я</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Прізвище</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Результат</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Назва університету</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="AddPerson">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Swimming</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="NameEdit_2"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="SurnameEdit_2"/>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="Universities_2">
            <property name="minimumSize">
             <size>
              <width>188</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>NULP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LNU</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="ResultEdit_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Result:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>University:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="minimumSize">
           <size>
            <width>751</width>
            <height>0</height>
           </size>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>188</number>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Ім'я</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Прізвище</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Результат</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Назва університету</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="AddPerson_2">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Run</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLineEdit" name="NameEdit_3"/>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="Universities_3">
            <property name="minimumSize">
             <size>
              <width>189</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>NULP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LNU</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="SurnameEdit_3"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="ResultEdit_3"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Surname:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Result:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>University:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="AddPerson_3">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="minimumSize">
           <size>
            <width>751</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>188</number>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Ім'я</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Прізвище</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Результат</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Назва університету</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>190</y>
          <width>261</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="1">
          <widget class="QLabel" name="JumpsClass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Jumps class name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="SwimmingClass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Run class name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="RunClass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Swimming class name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="SetClassName">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>749</width>
          <height>78</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="1">
          <widget class="QLineEdit" name="NameOFUniversityEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="SetName">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Date of holding:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Mounth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="spinBox_3">
             <property name="minimum">
              <number>2019</number>
             </property>
             <property name="maximum">
              <number>2050</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Day</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="spinBox_2">
             <property name="maximum">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>31</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="SetDate">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Name of universiade: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>200</y>
          <width>160</width>
          <height>93</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="1" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>LNU</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>NULP</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="LNU">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="SetWinner">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="NULP">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Amount of winners</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
